### DISCUSSION TITLE
YouTube + Firefox + uBlock Origin Slowness Issues

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2972] [General]
Hello everyone,

I am sure by now at least some people here have seen or heard about this issue.

Google/YouTube is targeting Firefox users who also use uBO and other content blockers on YouTube. Google has even admitted to doing this on purpose.

Ghacks article on this situation: https://www.ghacks.net/2023/11/20/youtube-video-loading-delayed-fix-inside/

Mental Outlaw video on this situation: https://www.youtube.com/watch?v=v4gXhmzQztE

Windows, computers and Technology video on this situation: https://www.youtube.com/watch?v=4RrZvg9z4t0

My 2 questions are:

A.) Can this situation be addressed in uBO code changes itself or through filter list modifications?

B.) Can this situation be on a watch list for filter list authors to make changes as needed?

**Comment by Yuki2718** [comment]
The thing is, it happens even *without any blocker* as long as you're chosen for the A/B test. We usually don't fix *site's own blunder*.

**Comment by TimTheBig** [comment]
use: `www.youtube.com##+js(nano-stb, resolve(1), *, 0.001)`
